src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+export const UserInfo: React.FC<{ todo: Todo }> = ({ todo }) => (
+  <>
+    {todo.user && (
```

Comment:
better to make this check in TodoInfo component
```
{ todo.user && <UserInfo user={todo.user} />}

Reply:
> better to make this check in TodoInfo component
> 
> ```
> { todo.user && <UserInfo user={todo.user} />} 
> ```
func foundUser returns User || null
and type conflicts apear if don't do this check in UserInfo FC